ops/pebble.py

```diff
@@ -2213,6 +2217,20 @@ def _request_raw(
 
         return response
 
+    # This matches the function of the same name in ops.log. It's repeated here,
+    # because ops.pebble doesn't know anything about ops.
+    def _security_event(self, event: str, *, level: str, description: str):
```

Comment:
I wonder if it'd be better to pass in a logging callable to `Client()`. Like `Client(log_security_event=log._security_event)` or `Client()` to mean no logging (the current default). Otherwise we're adding a bunch of logging to people who are importing Ops just to use the pebble Client, which is not exactly a breaking change, but probably annoying.